### PR TITLE
fix: aws multi

### PR DESCRIPTION
**Pull request by jokestax**
This PR
- Fix: change suffix for principalARN from `cluster name` to management's cluster name
- Update argo-cd manifest to v1.1.1 which includes resource health check for Crossplane Workspace resource.

### CODE REVIEW

File: aws-gitlab/templates/mgmt/components/argocd/argocd-cm.yaml [hunk 1]
```diff
@@ -11,34 +11,3 @@ data:
...
-              hs.status = "Degraded"
-            end
-            hs.message = installed_message .. " " .. healthy_message
-            return hs
-          end
-        end
-        
-        hs.status = "Progressing"
-        hs.message = "Waiting for wworkspace to finish running"
-        return hs
```
Comment (patrickdappollonio) [inline aws-gitlab/templates/mgmt/components/argocd/argocd-cm.yaml:44]: Curious to know why you're dropping these health checks

File: aws-github/templates/mgmt/components/argocd/argocd-cm.yaml [hunk 2]
```diff
@@ -10,35 +10,4 @@ data:
...
-              hs.status = "Degraded"
-            end
-            hs.message = installed_message .. " " .. healthy_message
-            return hs
-          end
-        end
-        
-        hs.status = "Progressing"
-        hs.message = "Waiting for wworkspace to finish running"
-        return hs
```
Comment (patrickdappollonio) [inline aws-github/templates/mgmt/components/argocd/argocd-cm.yaml:44]: Curious to know why you're dropping these health checks

File: aws-gitlab/templates/mgmt/components/argocd/argocd-cm.yaml (hunk 1 above)
Comment (muse-sisay) [reply aws-gitlab/templates/mgmt/components/argocd/argocd-cm.yaml:44]: Moved it to a centralized place here https://github.com/konstructio/manifests/blob/a33909601c3c8a0675f81e791aa54989bc732c08/argocd/cloud/argocd-cm.yaml#L26-L54

**Review by CristhianF7 — APPROVED** [review]

File: aws-github/templates/mgmt/components/argocd/argocd-cm.yaml (hunk 2 above)
Comment (muse-sisay) [reply aws-github/templates/mgmt/components/argocd/argocd-cm.yaml:44]: Moved it to a centralized place here https://github.com/konstructio/manifests/blob/a33909601c3c8a0675f81e791aa54989bc732c08/argocd/cloud/argocd-cm.yaml#L26-L54

**Pull request merged**